packages/core/src/hooks/useStarkAddress.ts

```diff
@@ -63,19 +66,22 @@ function queryKey({
   name,
   contract,
   network,
+  chainId,
 }: {
   name?: string;
   contract?: string;
   network?: string;
+  chainId?: bigint
```

Comment:
Notice that the hooks were already using a `network` argument, which contains the network name (double check because I may be wrong). Replace the code to only use `chainId` and no `network` anymore.